src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -306,6 +328,32 @@ export class CourseExerciseDetailsComponent extends AbstractScienceComponent imp
         }
     }
 
+    private mergeAndSortHistoryLists(sortedResults: Result[], selfLearningFeedbackRequests: SelfLearningFeedbackRequest[]): (Result | SelfLearningFeedbackRequest)[] {
```

Comment:
Could you rename the helper method to something like `mergeAndSortSelfLearningFeedbackRequestHistoryLists`? The current name might be too vague, or what do you think?